public/javascripts/components/weather.js

```diff
@@ -50,12 +50,12 @@ class Weather extends Component {
                 <td>{Math.round(precipProbability * 100)}%</td>
               </tr>
               <tr>
-                <th>wind</th>
-                <td>{windSpeed} mph {degreesToDirection(windBearing + 180)}</td>
+                <th>snowfall</th>
```

Comment:
Can we use `precipType` here?

Reply:
Pasting screenshot here cuz hangouts isn't uploading...here is an option putting it in the center...though it sounds like maybe you are thinking it's ok to leave it on the right?

![image](https://user-images.githubusercontent.com/29530398/54254709-6eafec00-451a-11e9-87cd-4118688eb904.png)

Reply:
Sorry, that was the wrong spot to paste this...wasn't paying attention.

Reply:
We could repurpose the allergies section (which no longer works), e.g.:

<img width="802" alt="Screen Shot 2019-03-12 at 10 09 01 PM" src="https://user-images.githubusercontent.com/126461/54255041-8768d380-4513-11e9-957c-d445a71e6d22.png">